cmd/jimmsrv/service/service.go

```diff
@@ -213,6 +217,10 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 // monitoring all changes to models. WatchControllers finishes when the
 // given context is canceled, or there is a fatal error watching models.
 func (s *Service) WatchControllers(ctx context.Context) error {
+	if s.jimm == nil {
```

Comment:
I wouldn't add all these checks. If jimm is nil and this panics we should see that from our tests. If it's happening in production I'd rather have the stack trace of where we tried to use a nil pointer, because this is an unrecoverable error. With this change all we get is an error messsage, "jimm not created" which is arguably worse because we can't see where it happened.